dask/bytes/tests/test_s3.py

```diff
@@ -193,32 +201,32 @@ def test_getsize(fmt):
 
 def test_modification_time_read_bytes():
     with s3_context('compress', files) as s3:
-        _, a = read_bytes('compress/test/accounts.*', s3=s3)
-        _, b = read_bytes('compress/test/accounts.*', s3=s3)
+        _, a = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
+        _, b = read_bytes('compress/test/accounts.*', storage_options={'s3': s3})
```

Comment:
Ahh, I see your point now! I didn't think of them like you do. However, I don't think that `read_bytes` is 'location-only' function. Let's see at S3 implementation:

```python
def read_bytes(path, s3=None, delimiter=None, not_zero=False, blocksize=2**27,
               sample=True, compression=None, **s3_params):
    if s3 is None:
        s3 = S3FileSystem(**s3_params)
```

You can easily notice that there are only `s3` and `**s3_params` arguments used as location options, while there are quite a few format options.